src/tribol/geom/GeomUtilities.hpp

```diff
@@ -361,6 +457,26 @@ FaceGeomError Intersection2DPolygonEnzyme( const RealT* xA, const RealT* yA, int
 
 #endif
 
+/*!
+ *
+ * \brief computes the segment overlap between two linear edges projected onto the same 2D plane
+ *
+ * \param [in] pX1 x-coordinates of edge 1 as projected onto a common plane
+ * \param [in] pY1 y-coordinates of edge 1 as projected onto a common plane
+ * \param [in] pX2 x-coordinates of edge 2 as projected onto a common plane
+ * \param [in] pY2 y-coordinates of edge 2 as projected onto a common plane
+ * \param [in] nV1 number of vertices on edge 1
+ * \param [in] nV2 number of vertices on edge 2
+ * \param [in,out] overlapX pointer to x coordinates of overlapping segment
+ * \param [in,out] overlapY pointer to y coordinates of overlapping segment
+ * \param [in,out] area overlap area/length
```

Comment:
```suggestion
 * \param [out] overlapX pointer to x coordinates of overlapping segment
 * \param [out] overlapY pointer to y coordinates of overlapping segment
 * \param [out] area overlap area/length
```

In general, check for `[in, out]`s that should be `[out]`s.  Here is doxygen documentation describing param: https://www.doxygen.nl/manual/commands.html#cmdparam

Reply:
I addressed them in this file. A more comprehensive check should be in another PR.